froide/helper/fields.py

```diff
@@ -30,13 +30,24 @@ def validate_svg(f):
     return f
 
 
+class SVGAndImageFieldInput(forms.ClearableFileInput):
```

Comment:
There's nothing SVG specific about it right? We also have ProfilePhotoFileInput with an accompanying template in the codebase that is basically identical and can be replaced with this helper. Maybe this input widget should move to helper/widgets.py for consistency.